scenarios/INTERACTION/merging/scenario.py

```diff
@@ -1,12 +1,21 @@
+import math
 from pathlib import Path
 
 from smarts.sstudio import types as t
 from smarts.sstudio import gen_scenario
 
 
+traffic_histories = [
+    t.TrafficHistoryDataset(
+        name=f"traffic_history_{hd}",
+        source_type="INTERACTION",
+        input_path=None,  # for example: f"./traffic_history_{hd}.txt"
```

Comment:
Is this for when users want to build the scenario without overwriting the history dataset?

Reply:
This is just because we aren't including the dataset itself in our repo, but we do want to include the `scenario.py`.  Our Makefile which is used by our CI does `scl scenario build-all ...`, so we need a way to not have it fail when the dataset isn't present.  So that's why we need the `None` option.